html-aam/roles-generic.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<html>
+<head>
+  <title>HTML-AAM Generic Role Verification Tests</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+
+<p>Tests ONLY the <code>generic</code> mappings defined in <a href="https://w3c.github.io/html-aam/">HTML-AAM</a>. Most test names correspond to a unique ID defined in the spec.<p>
+
+<p>These should remain in alphabetical order by tagname.</p>
+
```

Comment:
Adding some notes to ease reviewers' understanding of the PR.

All of the tests in this new file were preexisting but moved from ./html-aam/roles.html

This file tests all the HTML elements that are expected to end up with a `generic` role by default. It needs to call a different JS helper function to account for the implementation variance ("generic, "", or "none") that the ARIA WG deemed acceptable for this purpose.

Reply:
@cookiecrook Is there any documentation on this different browser treatment? Curious why an element with `generic` role may be mapped to `none` by some browsers. Is it accurate to say that generic and none are semantically equivalent?

Reply:
Looks like you answered below that this is a Chromium approach...

Reply:
> @cookiecrook Is there any documentation on this different browser treatment? Curious why an element with generic role (per spec) could be mapped to none by some browsers. Is it accurate to say that generic and none are semantically equivalent?

No. These had previously been mere implementation details, and those implementation-specific differences predated the inclusion of the `generic` role in the spec. The WG agreed that, until the implementations change to align on their own, or there is some other compelling reason to align on the returned "generic" role, there is little user benefit (an even some user risk, in the form of introduced regressions) to asking the implementations to "fix" this minor difference.